snap_ci/development_clean.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# bail out on errors
+set -e
+
+# declare function for EXIT trap
+function cleanup {
+    echo "Destroying droplet..."
+    vagrant destroy development -f
+}
+
+# Ensure that DigitalOcean droplet will be cleaned up
+# even if script errors (e.g., if serverspec tests fail).
+trap cleanup EXIT
+
+# If the previous build in snap-ci failed, the droplet
+# will still exist. Ensure that it's gone with a pre-emptive destroy.
+cleanup
+
+# Up the host in a separate command to avoid snap-ci command timeouts.
```

Comment:
After much testing, looks like the timeouts only trigger when STDOUT is silent for 10 minutes. As long as a processing is printing status updates, it should be fine. Still, it's better to deploy the droplet in a separate step from provisioning for help in debugging failed tests (e.g. our playbook broke vs. performance problems in a cloud provider).